### PR TITLE
Misc updates to the README

### LINKED ISSUE #5110: Update README
* the licensing model should be explained
 * the usage and examples needs checked and either updated or removed (and linked to inspec.io)
 * The 4 gems this repo builds should be explained (already raised as https://github.com/inspec/inspec/issues/4011)
 * @chris-rock and @arlimus should be added to the Kudos section
 * the copyright section should be updated

Aha! Link: https://chef.aha.io/features/SH-2356

### PR DESCRIPTION
**Pull request by clintoncwolfe**
## Description
Misc updates to the README

## Related Issue

Fixes #5110

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -366,6 +360,8 @@ You may also [browse the Supermarket for shared Compliance Profiles](https://sup
 
 ## Kudos
 
+Chef InSpec was originally created by Christoph Hartmann (@chris-rock) and Dominik Richter (@arlimus).
```
Comment (Vasu1105) [inline README.md]: I think those references should be in the [] brackets.

Comment (kagarmoe) [inline README.md]:
```suggestion
Chef InSpec was originally created by Christoph Hartmann and Dominik Richter.
```

Those are Slack handles, but Chris and Dominik have moved on to different companies.

File: README.md [hunk 2]
```diff
@@ -78,7 +78,9 @@ curl https://omnitruck.chef.io/install.sh | sudo bash -s -- -P inspec
 
 ### Install it via rubygems.org
 
-When installing from source, gem dependencies may require ruby build tools to be installed.
+When installing from source, gem dependencies may require ruby build tools to be installed. (A compiler-free variant is available with reduced functionality; use `inspec-core-bin` and `inspec-core`.)
```
Comment (kagarmoe) [inline README.md]:
```suggestion
Installing the Chef InSpec package from source may require installing ruby build tools to manage gem dependencies. (A compiler-free variant is available with reduced functionality; use `inspec-core-bin` and `inspec-core`.)
```

Changes for active voice.
I'm not sure about word 'manage' and I'm open to suggestions. Handle? Build?

**Review by kagarmoe — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (clintoncwolfe) [reply README.md]: They are also their github IDs.

**Pull request merged**